code/src/components/HeartButtons.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+
+export const HeartButtons = ({ id, hearts }) => {
+	const [ like, setLike ] = useState(hearts);
+	let [ yourLikes, setYourLikes ] = useState(0);
+
+	const [ yourPreviousLikes, setYourPreviousLikes ] = useState(JSON.parse(localStorage.getItem(id)) + 0);
+
+	localStorage.setItem(id, JSON.stringify(yourPreviousLikes));
+
+	const clickLike = (event) => {
```

Comment:
In case you didn't no - you don't need the parentheses when you only have one argument 
```suggestion
	const clickLike = event => {
```